lib/round.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+class Round
+  attr_accessor :deck,
+                :turns,
+                :turn,
+                :current_card,
+                :number_correct,
+                :correct_by_category,
+                :percent_correct
```

Comment:
There are an awful lot of `attr_readers` here. Typically we'd only have an `attr_reader` for 1) attributes that we initialize with and 2) that we need to access from outside the class. Some of these should be defined as their own methods below to follow ruby convention.